doc/api/next_api_changes/deprecations/20585-EP.rst

```diff
@@ -0,0 +1,10 @@
+RectangleSelector and EllipseSelector
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+``maxdist`` is deprecated, use ``handle_grab_distance`` argument instead
```

Comment:
I am quite happy with `handle_grab_distance` that you introduced in #20558! :)

Reply:
I'm not. While it's not really bad, the `handle` part does not add much information / make it more specific (there's nothing else you could grab). `handle` just makes the name longer. Also, `range` is more appropriate than `distance` because it explicitly includes all values (otherwise the distance should rather be `max_distance`, but that's quite long and not better than `range`).